src/main/java/core/basesyntax/OpationStrategy/OperationStrategyImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.OpationStrategy;
```

Comment:
There is a typo in the package name 'OpationStrategy'. It should likely be 'OperationStrategy'. Please correct the package name to ensure consistency and proper functionality within the project.